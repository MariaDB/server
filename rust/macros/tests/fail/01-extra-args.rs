include!("../include.rs");

register_plugin! {
    TestPlugin,
    ptype: PluginType::MariaEncryption,
    name: "debug_key_management",
    author: "Trevor Gross",
    description: "Debug key management plugin",
    license: License::Gpl,
    maturity: Maturity::Experimental,
    version: "0.1",
    init: TestPlugin,
    encryption: false,
    extra: false
}

fn main() {}
