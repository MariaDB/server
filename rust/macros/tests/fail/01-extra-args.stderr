error: unexpected field 'extra'
  --> tests/fail/01-extra-args.rs:14:5
   |
14 |     extra: false
   |     ^^^^^
